HCSP.js

```diff
@@ -0,0 +1,228 @@
+{
+    "translatorID": "3e375db7-5f6e-4861-ab8b-3ddb0f4accc9",
+    "label": "HCSP",
+    "creator": "Joris Muller",
+    "target": "http://www.hcsp.fr",
```

Comment:
```^https?//www\.hcsp\.fr/```
Make more efficient, allow for a current or future https version, escape ```.``` and end target with a ```/``` to work with proxies.